valohai_cli/commands/yaml/step.py

```diff
@@ -25,8 +26,11 @@ def step(filenames: List[str]) -> None:
     :param filenames: Path(s) of the Python source code files.
     """
     project = get_project()
-    assert project
-    config_path = project.get_config_filename()
+
+    if project is None:
+        info("no project linked - assuming files are in current directory.")
```

Comment:
Can you try if this works with just

```suggestion
    if project is None:
        info("no project linked - assuming files are in current directory.")
        project = Project(data={}, directory=os.getcwd())
```

i.e. you simulate a project that would live in this directory?

That way you don't need to change everything to accept no Project.